### ISSUE TITLE
Add a new document for explain Operator CRD settings

### ISSUE THREAD

**Issue by jmolmo** [rook/rook#4426]
**Is this a bug report or feature request?**
* Feature Request

**What should the feature do:**
Create a new document to explain the different settiong and options available in the Operator CRD

**What is use case behind this feature:**
We do not have this document, and although we have most of the things explained in the CRD file, it could be interesting to have this information also as a part of the projects documentation. This will allow us to explain with more detail setttings, different configurations, etc ...

**Environment**:
Documentation: Installation using CRDs in K8s, Openshift and with Helm Charts in k8s

**Comment by ChanSays** [comment]
Hi can I take on this issue?

**Comment by travisn** [comment]
> Hi can I take on this issue?

@ChanSays Yes, thanks! We have just been documenting the settings directly in the operator.yaml, but it would really be better if we could convert that to the documentation. 

I'd suggest a new document be created with a header like this, which will add the new topic in the table of contents after "CSI Common Issues" and before "Monitor Health":
```
---
title: Operator Settings
weight: 11128
indent: true
---
```

**Comment by ChanSays** [comment]
thanks for reply!
 will look into asap

**Comment by prernasony** [comment]
@ChanSays Are you still working on it ?
